cfpb/cfgov-refresh
# Fix background color and text alignment in heroes with no image

Fixes https://GHE/CFGOV/platform/issues/3276

## Changes

- Move `background-color` rule outside of `if img` condition
- Always output `<div class="m-hero_image-wrapper">`, even if empty, for correct text alignment

## Testing

1. Create a Sublanding Page
2. Add a Hero
3. Set some text for the **Heading** and **Body**
4. Remove empty default link from the **Links** list
5. Add a valid hex color in the **Background color** field
6. Click Preview and see a hero with the specified background color and expected text alignment 

## Checklist


## Testing checklist

### Browsers
